avocado/core/sysinfo.py

```diff
@@ -11,387 +11,21 @@
 # This code was inspired in the autotest project,
 # client/shared/settings.py
 # Author: John Admanski <jadmanski@google.com>
-
-import gzip
-import json
+import filecmp
 import logging
 import os
-import shlex
-import shutil
-import subprocess
 import time
-from abc import ABC, abstractmethod
 
-from ..utils import astring, genio
+from ..utils import genio
 from ..utils import path as utils_path
-from ..utils import process, software_manager
+from ..utils import software_manager
+from ..utils import sysinfo_collectible as sysinfo
```

Comment:
Yes, I am not sure with the name. Because it's not the whole sysinfo module, it is just collectibles, but I don't have any preference between `sysinfo_collectible` or `sysinfo`.

About package + module, I think it is not necessary to create new package when there will be just one module `collectible`.

Reply:
Ok, so my suggestion would be to remove the underscore and use sysinfo only.